### PR TITLE
[HDWWIZ] Update Polish (pl-PL) translation 

### PR DESCRIPTION
**Pull request by pithwz**
## Proposed changes
small update

### CODE REVIEW

**Comment by HBelusca** [comment]
Cc @reactos/lang-polish

File: dll/cpl/hdwwiz/lang/pl-PL.rc [hunk 1]
```diff
@@ -78,7 +79,7 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYS
 CAPTION "Kreator dodawania sprzętu"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Jeśli nie widzisz odpowiedniej kategorii urządzenia kliknij przycisk Pokaż wszystkie urządzenia.", IDC_STATUSTEXT, 21, 9, 275, 18
+    LTEXT "Jeśli nie widzisz odpowiedniego sprzętu, kliknij przycisk Pokaż wszystkie urządzenia.", IDC_STATUSTEXT, 21, 9, 275, 18
```
Comment (Saibamen) [inline dll/cpl/hdwwiz/lang/pl-PL.rc:82]: From Win 10:
```suggestion
    LTEXT "Jeśli nie widzisz odpowiedniej kategorii sprzętu, kliknij przycisk Pokaż wszystkie urządzenia.", IDC_STATUSTEXT, 21, 9, 275, 18
```

File: dll/cpl/hdwwiz/lang/pl-PL.rc [hunk 2]
```diff
@@ -78,7 +79,7 @@ STYLE DS_SHELLFONT | DS_MODALFRAME | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYS
...
     LTEXT "&Popularne typy urządzeń:", IDC_HWTYPESTEXT, 21, 31, 180, 8
```
Comment (Saibamen) [inline dll/cpl/hdwwiz/lang/pl-PL.rc:83]: From Win 10:
```suggestion
    LTEXT "&Popularne typy sprzętu:", IDC_HWTYPESTEXT, 21, 31, 180, 8
```

**Review by Saibamen — CHANGES_REQUESTED** [review]

File: dll/cpl/hdwwiz/lang/pl-PL.rc (hunk 1 above)
Comment (HBelusca) [reply dll/cpl/hdwwiz/lang/pl-PL.rc]: ReactOS does not aim to have a 100% copy-paste of the translations.

**Review by Saibamen — APPROVED** [review]
OK

**Pull request merged**